NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -224,6 +225,30 @@ def is_shower_primary(particle):
     else:
         return False
 
+def check_path_to_tables(config_file_path):
+    """
+    Checks if the paths for the PROPOSAL tables in the input config file are
+    existing directories.
+    """
+
+    with open(config_file_path, 'r') as f:
+
+        config_file = json.load(f)
+
+    path_to_tables = config_file['global']['interpolation']['path_to_tables']
+    path_to_tables_readonly = config_file['global']['interpolation']['path_to_tables_readonly']
+
+    if not os.path.isdir(path_to_tables):
+
+        error_msg  = "'path_to_tables' in {} points to a non-existing directory. ".format(config_file_path)
```

Comment:
could you also print the currently set path, same below

Reply:
You're right, that would be better.